app.rb

```diff
@@ -0,0 +1,106 @@
+require_relative './book'
+require_relative './person'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+
+class App
+  def show_books
+    if Book.all.empty?
+      puts 'There\'s no books added yet!'
+    else
+      Book.all.each do |book|
+        puts %(Title: "#{book.title}", Author: #{book.author})
+      end
+    end
+  end
+
+  def show_people
+    if Student.all.empty? && Teacher.all.empty?
+      puts 'There\'s no people added yet'
+    else
+      Student.all.each do |student|
+        puts %([Student] Name: #{student.name}, ID: #{student.id}, Age: #{student.age})
+      end
+      Teacher.all.each do |teacher|
+        puts %([Teacher] Name: #{teacher.name}, ID: #{teacher.id}, Age: #{teacher.age})
+      end
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    selection = gets.chomp.to_i
+    if [1, 2].include?(selection)
+      print 'Age: '
+      age = gets.chomp
+      print 'Name: '
+      name = gets.chomp
+      case selection
+      when 1
+        print 'Classroom: '
+        classroom = gets.chomp
+        print 'Has parent permission? [Y/N]: '
+        permission = gets.chomp
+        Student.new(age, classroom, name: name, parent_permission: permission)
+        puts 'Student created!'
+      when 2
+        print 'Specialization: '
+        specialization = gets.chomp
+        Teacher.new(age, specialization, name: name)
+        puts 'Teacher created!'
+      end
```

Comment:
- [ ] I get the below error when I try to create a new teacher. Please kindly fix this error so that a new teacher can be created

![image](https://user-images.githubusercontent.com/43995830/211178565-76aa079d-230e-4a22-9caf-3e2de151b8fd.png)

Reply:
Thank you so much for your kind observation! I will get it fixed asap!